docs/tools/index.txt

```diff
@@ -9,17 +9,65 @@ This section of the documentation covers some of the tools available. If you
 know of any other tools that work well with CrateDB, please `get in touch
 <mailto:office@crate.io>`_.
 
-.. rubric:: Table of Contents
+.. SEEALSO::
 
-.. toctree::
-
-   management
-   dataviz
+    The `client tools`_ category on our blog.
 
 .. toctree::
    :hidden:
 
    streamsets
 
+Data Visualization
+==================
+
+Pentaho
+-------
+
+Pentaho prepares and blends data, delivering business analytics from any
+source. You can connect to CrateDB clusters by using the standalone version of
+`our JDBC driver <https://crate.io/docs/reference/jdbc/>`_.
+
+-  `Website <http://community.pentaho.com>`__
+-  `Github <https://github.com/pentaho/pentaho-kettle>`__
+
+StreamSets Data Collector
+-------------------------
+
+StreamSets Data Collector is a lightweight, powerful engine that streams data
+in real time. Use Data Collector to route and process data in your data
+streams. Read the manual (:doc:`streamsets`) on how to use CrateDB as an origin
+and destination in a StreamSets pipeline.
+
+-  `Download <https://streamsets.com/opensource/>`_
+-  `Official documentation
+   <https://streamsets.com/documentation/datacollector/latest/help/>`__
+
+Zoomdata
+--------
+
+Zoomdata is a SaaS solution for deep analysis of big data sets in a visual
+interface. CrateDB is available as a data source connection.
+
+Grafana
+-------
+
+Grafana is a very versatile tool for visualizing all kinds of time series data,
+and it has a CrateDB plugin.
+
+-  `Website <https://grafana.net/>`__
+-  `CrateDB Datasource Plugin
+   <https://grafana.net/plugins/crate-datasource>`_
+
+Metabase
+--------
+
+Metabase is an open source, easy-to-use visualization that answers questions
+about your data. It's written in Clojure and comes with a CrateDB driver out of
+the box!
```

Comment:
maybe "." (dot) instead of "!" ?

Reply:
was just copying this info across. will be removed soon